addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -2176,4 +2180,31 @@ MockServer.include({
             starred_counter: this._getRecords('mail.message', [['starred_partner_ids', 'in', user.partner_id]]).length,
         };
     },
+    /**
+     * Simulate the `notify_cancel_by_type` on `mail.thread` .
+     * Note that this method is overridden by snailmail module but not simulated here.
+     */
+    _mockMailThreadNotifyCancelByType(model, notificationType) {
+        // Query related mail messages and notifications
+        const notifMails = this._getRecords('mail.notification', [
```

Comment:
Suggestion: simply call it `notifications`. I have no idea where that mails is coming from.